pages.zh/linux/dmesg.md

```diff
@@ -0,0 +1,32 @@
+# dmesg
+
+> 显示或控制内核环形缓冲区。
+> 更多信息：<https://man7.org/linux/man-pages/man1/dmesg.1.html>。
```

Comment:
We're working on transitioning from man7.org to manned.org (#5790)
```suggestion
> 更多信息：<https://manned.org/dmesg>。
```

Reply:
Okay, I will use this link in the future